src/SpeedDial/SpeedDial.js

```diff
@@ -0,0 +1,128 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import classNames from 'classnames';
```

Comment:
I try to sort the import by scope. Maybe you would want to do the same and to keep it before the material-ui imports.